Lib/test/test_asyncio/test_events.py

```diff
@@ -2653,33 +2653,23 @@ def callback(*args):
 
         h1 = asyncio.TimerHandle(when, callback, (), self.loop)
         h2 = asyncio.TimerHandle(when, callback, (), self.loop)
-        # TODO: Use assertLess etc.
-        self.assertFalse(h1 < h2)
-        self.assertFalse(h2 < h1)
-        self.assertTrue(h1 <= h2)
-        self.assertTrue(h2 <= h1)
-        self.assertFalse(h1 > h2)
-        self.assertFalse(h2 > h1)
-        self.assertTrue(h1 >= h2)
-        self.assertTrue(h2 >= h1)
-        self.assertTrue(h1 == h2)
-        self.assertFalse(h1 != h2)
+
+        self.assertLessEqual(h1, h2)
+        self.assertLessEqual(h2, h1)
+        self.assertGreaterEqual(h1, h2)
+        self.assertGreaterEqual(h2, h1)
+        self.assertEqual(h1, h2)
 
         h2.cancel()
-        self.assertFalse(h1 == h2)
+        self.assertNotEqual(h1, h2)
 
         h1 = asyncio.TimerHandle(when, callback, (), self.loop)
         h2 = asyncio.TimerHandle(when + 10.0, callback, (), self.loop)
-        self.assertTrue(h1 < h2)
-        self.assertFalse(h2 < h1)
-        self.assertTrue(h1 <= h2)
-        self.assertFalse(h2 <= h1)
-        self.assertFalse(h1 > h2)
-        self.assertTrue(h2 > h1)
-        self.assertFalse(h1 >= h2)
-        self.assertTrue(h2 >= h1)
-        self.assertFalse(h1 == h2)
-        self.assertTrue(h1 != h2)
+        self.assertLess(h1, h2)
+        self.assertGreater(h2, h1)
+        self.assertLessEqual(h1, h2)
```

Comment:
Keep different tests for less and less equal

Reply:
I'm not quite sure what you want. The original test are `assertTrue(h1 < h2)` and `assertTrue(h1 <= h2)` which should be equivalent to `assertLess(h1, h2)` and `assertLessEqual(h1, h2)`.

Reply:
This comment for meant for line 2657 which missed assertLess, seems like Github messed up the comments.

Reply:
Oh. Ok makes sense now!

Reply:
@kumaraditya303

`self.assertLess(h1, h2)` gives false for line 2654-2655, it makes sense to use `self.assertLess(h1, h2)` for 2666-2667

Reply:
@kumaraditya303

`self.assertLess(h1, h2)` gives false for line 2654-2655, it makes sense to use `self.assertLess(h1, h2)` for 2666-2667